config/settings/segmentedtickerplant.q

```diff
@@ -0,0 +1,51 @@
+// Segmented TP config
+
+\d .stplg
+  
+multilog:`tabperiod;            // [tabperiod|none|periodic|tabular|custom]
+multilogperiod:0D01;
+errmode:1b;
+batchmode:`defaultbatch;        // [memorybatch|defaultbatch|immediate]
+customcsv:hsym first .proc.getconfigfile["stpcustom.csv"];
+replayperiod:`day               // [period|day|prior]
+
+\d .proc
+
+loadprocesscode:1b        
+tptype:`segmented
```

Comment:
is this variable required? I'm not sure it is used

Reply:
Currently this variable is used in the subscriptions.q code to determine the subscription method.

Reply:
This is needed in the STP but was unnecessary in subscriptions.q

Reply:
Is this variable used or not? Is it actually used as .proc.tptype?